cmd/admin.go

```diff
@@ -277,10 +283,29 @@ func runChangePassword(c *cli.Context) error {
 }
 
 func runCreateUser(c *cli.Context) error {
-	if err := argsSet(c, "name", "password", "email"); err != nil {
+	if err := argsSet(c, "name", "email"); err != nil {
 		return err
 	}
 
+	if c.IsSet("password") && c.IsSet("random-password") {
+		return errors.New("cannot set both -random-password and -password flags")
+	}
+
+	var password string
+
+	if c.IsSet("password") {
+		password = c.String("password")
+	} else if c.IsSet("random-password") {
+		password, err := generate.GetRandomString(30)
```

Comment:
is it really reasonable to generate 30 characters long password?

Reply:
Maybe give another command line argument `password length`.

Reply:
@lafriks my passwords are usually 40 - 75 characters long ( including characters ), I probably added that based on what I'd use. 😃😃😃

@lunny nice enough? What do you think @lafriks?

Reply:
Option to specify length would be great. Default should probably be something like 12